ArcGISRuntimeSDKQt_CppSamples/Maps/GenerateOfflineMap_Overrides/GenerateOfflineMap_Overrides.qml

```diff
@@ -0,0 +1,126 @@
+// [WriteFile Name=GenerateOfflineMap, Category=Maps]
+// [Legal]
+// Copyright 2017 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 2.2
+import QtQuick.Dialogs 1.2
+import QtQuick.Window 2.2
+import Esri.Samples 1.0
+import Esri.ArcGISExtras 1.1
+import Esri.ArcGISRuntime.Toolkit.Dialogs 100.4
+
+GenerateOfflineMap_OverridesSample {
+    id: offlineMapOverridesSample
+    clip: true
+    width: 800
+    height: 600
+
+    property real scaleFactor: (Screen.logicalPixelDensity * 25.4) / (Qt.platform.os === "windows" ? 96 : 72)
```

Comment:
change to handle linux = 96

Reply:
Good catch @lsmallwood .  I believe I used `Qt.platform.os === "windows" ||  Qt.platform.os === "linux" ? 96 : 72` in other files